docs/installation/presto-cli-installation.rst

```diff
@@ -9,18 +9,17 @@ The Presto CLI provides a terminal-based interactive shell for running queries.
 Download the `presto-cli` and copy it to the location you want to run it from. This location may be any node that has network access to the coordinator. Then rename it to presto and make it executable with chmod +x:
 ::
 
- $ mv presto-cli-0.115-executable.jar presto
+ $ mv presto-cli-*-executable.jar presto
  $ chmod +x presto
 
 .. NOTE:: Presto must run with Java 8, so if Java 7 is the default on your cluster, you will need to explicitly specify the Java 8 executable. For example, ``<path_to_java_8_executable> -jar presto``. It may be helpful to add an alias for the Presto CLI: ``alias presto='<path_to_java_8_executable> -jar <path_to_presto>'``.
 
-By default, ``presto-admin`` installs the TPC-H connector for you, which generates TPC-H data on-the-fly.  Using this connector, issue the following commands to run your first Presto query:
+By default, ``presto-admin`` installs the TPC-H connector, which generates TPC-H data on-the-fly.  Using this connector, issue the following commands to run your first Presto query:
```

Comment:
"Installs" suggests that pa is somehow changing presto to add functionality that isn't there by default. Suggest s/installs/configures/